unfurl/src/jvmMain/kotlin/Platform.jvm.kt

```diff
@@ -0,0 +1,9 @@
+package me.saket.unfurl
+
+import io.ktor.client.engine.HttpClientEngine
+import io.ktor.client.engine.okhttp.OkHttp
+
+internal actual fun provideHttpClientEngine(): HttpClientEngine {
+  return OkHttp.create {
+  }
```

Comment:
Let's share the source sets for JVM and Android?

Reply:
Will do. Thanks